src/components/fab.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
+import plus from '../assets/plusIcon.svg';
+function Fab() {
```

Comment:
보통 import 문 아래 한 줄 띄어씁니다 

```suggestion
import plus from '../assets/plusIcon.svg';

function Fab() {
```

Reply:
Fab 이라는게 뭘 의미하는지 바로 알기가 어려웠어요.

너무 긴 단어는 줄여서 쓰기도 하지만 그럴 떄는 다른 사람도 이해할 수 있는 축약어인지도 고민해보시면 좋을 것 같아요 !